Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:14.04
+
+ADD ./WcaOnRails /app/WcaOnRails
+
+ENV agi="apt-get install --yes --no-install-recommends"
+
+# From https://www.brightbox.com/docs/ruby/ubuntu/
+RUN $agi software-properties-common
+RUN apt-add-repository --yes ppa:brightbox/ruby-ng
+
+RUN apt-get update
```

Comment:
`apt-get update` and `apt-get install` should be done in the same line, see https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get for explanation.

You can also make the size of the image smaller by combining several `RUN` commands, see https://www.ianlewis.org/en/creating-smaller-docker-images